### PR TITLE
Report

### PR DESCRIPTION
**Pull request by Hroy02**

### CODE REVIEW

File: src/main/java/bgu/spl/mics/application/passiveObjects/Report.java [hunk 1]
```diff
@@ -9,144 +9,161 @@
...
-	/**
+    private String _missionName;
+    private int _M;
+    private List<String> _agentsSerialNumbers;
+    private String _gadgetName;
+    private int _timeIssued;
+    private int _timeCreated;
+    private int _MoneyPenny;
+    private List<String> _agentsNames;
+    private int _qTime;
```
Comment (IdanTo1) [inline src/main/java/bgu/spl/mics/application/passiveObjects/Report.java]: Qtime

File: src/main/java/bgu/spl/mics/application/passiveObjects/Report.java [hunk 2]
```diff
@@ -9,144 +9,161 @@
...
+    public Report() {
+        _missionName = "";
+        _M = -1;
+        _agentsSerialNumbers = null;
+        _gadgetName = "";
+        _timeCreated = -1;
+        _timeIssued = -1;
+        _MoneyPenny = -1;
+        _agentsNames = null;
+        _qTime = -1;
```
Comment (IdanTo1) [inline src/main/java/bgu/spl/mics/application/passiveObjects/Report.java]: why is it needed at all?

**Review by IdanTo1 — CHANGES_REQUESTED** [review]
looks great! only little fix

File: src/main/java/bgu/spl/mics/application/passiveObjects/Report.java (hunk 1 above)
Comment (Hroy02) [reply src/main/java/bgu/spl/mics/application/passiveObjects/Report.java]: Done

File: src/main/java/bgu/spl/mics/application/passiveObjects/Report.java (hunk 2 above)
Comment (Hroy02) [reply src/main/java/bgu/spl/mics/application/passiveObjects/Report.java]: Changed and added a comment

**Review by IdanTo1 — APPROVED** [review]
perfect
you can merge it

**Pull request merged**